.env

```
@@ -1 +1,2 @@
-MONGODB_URI="mongodb://localhost/research-activities-db"
\ No newline at end of file
+MONGODB_URI="mongodb://localhost/research-activities-db"
+MONGODB_URI1="mongodb://heroku_1b8nz662:hpueci1942ulml4c9h1runn5qk@ds113826.mlab.com:13826/heroku_1b8nz662"
```

.env is only used in local environment 
no need to commit this change to the main branch
```suggestion
MONGODB_URI="mongodb://localhost/research-activities-db"
```

---

Yea forgot about that 
I fixed it